lib/net/pool.js

```diff
@@ -704,6 +720,81 @@ class Pool extends EventEmitter {
     this.compactBlocks.clear();
   }
 
+  /**
+   * Start the filters headers sync.
+   */
+
+  async startFilterHeadersSync() {
+    this.logger.info('Starting filter headers sync (%s).',
+      this.chain.options.network);
+    if (!this.opened || !this.connected)
+      return;
+
+    this.filterSyncing = true;
+    const startHeight = 1;
+    const chainHeight = await this.chain.tip.height;
+    console.log('chainHeight', chainHeight);
+    const stopHeight = chainHeight > 2000 ? 2000 : chainHeight;
+    const stopHash = await this.chain.getHash(stopHeight);
+    this.peers.load.sendGetCFHeaders(
+      common.FILTERS.BASIC,
+      startHeight,
+      stopHash);
+
+    this.getcfheadersFilterType = common.FILTERS.BASIC;
+    this.getcfheadersStopHash = stopHash;
+  }
+
+  /**
+   * Start the filters sync.
+   */
+
+  async startFilterSync() {
+    this.logger.info('Starting filter sync (%s).',
+      this.chain.options.network);
+    if (!this.opened || !this.connected)
+      return;
+
+    this.filterSyncing = true;
+    const startHeight = 0;
+    const stopHash = await this.chain.getHash(29);
+    this.peers.load.sendGetCFilters(
+      common.FILTERS.BASIC,
+      startHeight,
+      stopHash);
+    this.getcfiltersFilterType = common.FILTERS.BASIC;
+    this.getcfiltersStartHeight = startHeight;
+    this.getcfiltersStopHash = stopHash;
+  }
+
+  /**
+   * Start the headers sync using getHeaders messages.
+   * @private
+   * @return {Promise}
+   */
+
+  async startHeadersSync() {
+    if (!this.syncing)
+      return;
+    console.log('startHeadersSync');
+    let locator;
+    try {
+      locator = await this.chain.getLocator();
+    } catch (e) {
+      this.emit('error', e);
+      return;
+    }
+
+    const peer = this.peers.load;
+    if (!peer) {
+      this.logger.info('No loader peer.');
+      return;
+    }
+    this.chain.synced = false;
+    peer.sendGetHeaders(locator);
+    // this.emit('headersFull');
+  }
+
```

Comment:
Do we need this function? We already have `if (this.options.neutrino) {sendGetHeaders(locator)}` in `sendLocator()` which is called by `startSync()` so that seems redundant